spotify/src/main/resources/connector.xml

```diff
@@ -1,16 +1,32 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Already updated in current code: https://github.com/wso2/esb-connectors/blob/master/spotify/spotify-connector/spotify-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/connector.xml